### PR TITLE
added shell script for loop numbers

### PR DESCRIPTION
**Pull request by cloudtechmasters**
Added Shell script to loop numbers

### CODE REVIEW

**Comment by cloudtechmasters** [comment]
changes look good

**Comment by Venkatkeesara** [comment]
Looks good

**Comment by syedhashmi123** [comment]
Looks good to move forward

**Review by Venkatkeesara — COMMENTED** [review]
Looks good

**Comment by kotieswar49** [comment]
Good to go

**Review by kotieswar49 — APPROVED** [review]
Good to go

**Review by Venkatkeesara — COMMENTED** [review]
Code changes looks good

File: loopnumbers.sh
Comment (Vinodsiri94) [inline loopnumbers.sh:1]: Looks Good

**Review by syedhashmi123 — APPROVED** [review]
look good to move forward with

**Review by kotieswar49 — COMMENTED** [review]
Good to go

**Review by Venkatkeesara — COMMENTED** [review]
Looks good

**Review by Vinodsiri94 — COMMENTED** [review]
Looks Good

**Review by Venkatkeesara — APPROVED** [review]
looks good

**Review by Vinodsiri94 — APPROVED** [review]
Looks Good

**Pull request merged**